src/collective/easyform/browser/widgets.py

```diff
@@ -106,3 +112,19 @@ def __call__(self):
         if not css_class:
             return dflt
         return css_class
+
+
+PloneRenderWidget.index = ViewPageTemplateFile("widget.pt")
+logger.info("Patched plone.app.z3cform.views.RenderWidget to use our widget.pt.")
```

Comment:
This makes me uneasy. If a project has also customized this template using z3c.jbot, which one wins?

Reply:
It makes me uneasy too.
One way to solve this, would be to actually take over the widget template from easyform in plone.app.z3cform. Should be okay. But definitely not now, a few days before Plone 6 final. That makes me far more uneasy. ;-)